src/Boo.Lang.Compiler/Boo.Lang.Compiler.csproj

```diff
@@ -25,6 +26,7 @@
     <DefineConstants>TRACE;DEBUG;IGNOREKEYFILE,NET_2_0</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>

```

Comment:
What is this "Prefer32Bit" property that keeps popping up all over these project files?